mmv1/products/netapp/storagePool.yaml

```diff
@@ -25,9 +25,21 @@ description: |
  The capacity of the pool can be split up and assigned to volumes within the pool. Storage pools are a billable
  component of NetApp Volumes. Billing is based on the location, service level, and capacity allocated to a pool
  independent of consumption at the volume level.
+
+ Storage pools of service level Flex are available as zonal (single zone) or regional (two zones in same region) pools.
+ Zonal and regional pools are high-available within the zone. On top of that, regional pools have `replica_zone` as
+ hot standby zone. All volume access is served from the `zone`. If `zone` fails, `replica_zone`
+ automatically becomes the active zone. This will cause state drift in your configuration.
```

Comment:
I assume this is the best option at this point, but I'll still ask: is there any chance of this being modeled differently in the API, so that there is a separate "desired" zone and "actual" zone? This is what we usually recommend to prevent drift scenarios like this.